index.html

```diff
@@ -61,7 +69,7 @@ <h2 class="supporting-text">
         <h1 class="my-recent-works">
             My Recent Works
```

Comment:
- [ ] I kindly suggest that you ensure that your header and the horizontal line beside it do not overlap. Kindly see the implementation in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

| Your Design  | Figma Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52934128/179191364-b660d865-7325-4513-88c2-622ae9cf4d5a.png) | ![image](https://user-images.githubusercontent.com/52934128/179191323-425fa168-2306-434f-956f-a8ab252bce73.png)  |